### DISCUSSION TITLE
Looks like you're cooking something new?

### DISCUSSION THREAD

**Discussion by BMCwebdev** [Dwlad90/stylex-swc-plugin#43] [Q&A] [answered]
So with what I see in the ReadMe and in the other discussion here, you have a v0.3.0 coming out with a new compiler? Wondering if I should just wait for that to be released? Thanks!

**Comment by Dwlad90** [comment, marked as answer]
That's right, there will be a release with a [new compiler](https://github.com/Dwlad90/stylex-swc-plugin/discussions/39) soon, which includes the convenience of the babel plugin, the speed of SWC and access to the file system. 

Therefore, the previous version of the plugin is marked as deprecated, and the NextJS plugin is renamed to "@stylexswc/nextjs-swc-plugin". For now, support for the SWC plugin as a separate package will continue, but in the future it may be completely replaced by a new compiler.

**Reply by BMCwebdev** [reply]
That's exciting. Okay, I will keep my eyes out for your next release!
